Gems/Multiplayer/Code/Source/NetworkTime/NetworkTime.cpp

```diff
@@ -95,6 +105,7 @@ namespace Multiplayer
 
                     if (networkTransform != nullptr)
                     {
+                        // We're not presently factoring in interpolated position here
```

Comment:
Good point :\ luckily the error here should be much smaller than velocity * rtt, but we still might want to extrude our intersect volumes a bit to compensate.

Reply:
It's something I plan to test more once weapons are in and we increase the density of the test level. It's an awkward spot to access data to interpolate so I figured let's see what the specific edge cases are before we settle on handling.